src/main/java/org/folio/circulation/domain/RequestQueue.java

```diff
@@ -75,9 +75,12 @@ private boolean requestIsFulfillableByItem(Request request, Item item) {
     if (request.getRequestLevel() == RequestLevel.TITLE) {
       String itemInstanceId = item.getInstanceId();
       String requestInstanceId = request.getInstanceId();
+      if(request.isRecall() && request.isNotYetFilled()) {
+        return itemInstanceId != null && itemInstanceId.equals(requestInstanceId);
+      }
```

Comment:
```suggestion
        return requestInstanceId.equals(itemInstanceId);
      }
```
You can do it this way, as a request in a DB can't not have instance ID

Reply:
makes sense, thanks